### DISCUSSION TITLE
Looper State 4-6

### DISCUSSION THREAD

**Discussion by reynmusician** [KBLiveSolutions/open.control#22] [Troubleshooting] [answered]
Hi Pierre,

I'm using a lot of Loopers in my Ableton session. Six to be exact. (which is just perfect for the Open-Control). I'm using the leds of the Open-Control to show me the status of all loopers. I've added the "(LOOPER#)" to the names to all 6 loopers, but only the first 3 are showing. Sometimes after a restart I get 4 if I'm lucky, but never all 6.

Can you help me?

Cheers, Reyn

Instagram.com/reynmusician
Facebook.com/reynmusician
YouTube.com/reynmusician

**Comment by KBLiveSolutions** [comment, marked as answer]
Yes you're right, this was a bug.
Can you please [download](https://github.com/KBLiveSolutions/open.control-Remote-Script/archive/refs/heads/main.zip) and install the script again ?
It should work fine now.

Kind regards,
Pierre

**Reply by reynmusician** [reply]
Perfect! I'll update the script. Thanks Pierre.
